src/openbook/models.py

```diff
@@ -1,57 +1,94 @@
-from sqlalchemy import Column, String, Integer, ForeignKey
+import enum
+
+from sqlalchemy import Column, Enum, ForeignKey, Index, Integer, String
 from sqlalchemy.orm import declarative_base, relationship
 
 Base = declarative_base()
 
-class Reading_list_book(Base):
-    __titlename__ = 'reading_list_book'
-    book_id = Column(Integer, ForeignKey('users.id'), primary_key=True)
-    reading_list_id = Column(Integer, ForeignKey('reading_lists'), primary_key=True)
 
-    status = Column(String, default='reading')
+class BookStatus(enum.Enum):
+    """Enumeration for the status of a book in a user's list."""
+
+    RECOMMENDED = "recommended"
+    READING = "reading"
+    COMPLETED = "completed"
+
+
+class UserBook(Base):
+    """Represents the association between users and books."""
+
+    __tablename__ = "user_book"
```

Comment:
Be consistent in the table name: either all plural (`users_books`), or don't make the user table plural (`user`). Same with `books`.